chocolatey/choco
# (#2867) Add --online option to all commands

## Description Of Changes

This commit adds the --online option to all Chocolatey commands, to allow opening of the online help documentation for the command that is being executed.

For example, running:

choco install -h

will output the help documentation to the console, but running:

choco install -h --online

will not output anything to the command line, and instead open the default browser to the help documentation for the command.

## Motivation and Context

I want to improve the experience of reading the help documentation, by providing a --online option to open in default browser, similar to what is done by the `Get-Help` cmdlet in PowerShell.

## Testing

1. Compile code
2. Set debug options to be `install -h --online`
3. Run choco.exe and see that default browser opens to help page for install command
4. Run `install -h` and see that help is written to output
5. Run `bob -h --online` and see that command is not found

## Change Types Made


## Related Issue

Fixes #2867

## Change Checklist
